### ISSUE TITLE
failed transaction still showing as pending

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/giveth-dapps-v2#795]
testing on `staging.giveth.io`
clean cache and cookies
brave browser

I created a donation tx recently that I purposefully set to run out of gas - the tx failed.

however on my donations and my public profile it still shows the tx as pending but if you follow the provided link you can see the tx actually failed - the failed transaction should update the donation status to failed instead of remaining as pending forever.

![image.png](https://images.zenhubusercontent.com/5fff1c86d883b3d9a858ddbc/96513f92-7ba4-4c7d-b8c7-36623f14a1a8)

here's the tx hash - https://ropsten.etherscan.io/tx/0xb7ee3deebb4be29bec35b4055374d9aedb17fcc960503d1d43bc89a1f309935d

**Comment by RamRamez** [comment]
this issue is due to a problem with front-end and back-end. Front-end is related to issue #655 that is resolved in PR #814
for back-end @mohammadranjbarz is working on it.

**Comment by mohammadranjbarz** [comment]
@divine-comedian  @RamRamez  We deployed a new PR, can you test it again to make sure it's working fine?

**Comment by divine-comedian** [comment]
okay so I tried the same trick with making the tx fail with running out of gas. the tx now shows as failed on my donations page as well as the project page
![image](https://user-images.githubusercontent.com/67759413/171279276-75aefe48-2570-423e-8881-36149272f304.png)

some of my previous donations before the PR was made that are still pending on the UI have resolved as successful and some have failed, will the app look these txs up and resolve them on the UI or will they remain pending forever? @mohammadranjbarz

**Comment by mohammadranjbarz** [comment]
@divine-comedian  our job just check the pending donations, so if a donation by mistake got verified/failed the job will not change it unless we change the status manually to `pending` in DB
And for pending donations that are for a long time ago, the below issue would fix it https://github.com/Giveth/impact-graph/issues/510
so we can close this issue and check the thing that you said in the other issue
